src/puppetlabs/puppetdb/query/event_counts.clj

```diff
@@ -116,10 +116,11 @@
   "Helper function to transform the event count subject data from the raw format that we get back from the
   database into the more structured format that the API specifies."
   [summarize_by]
-  (fn [rows]
-    (mapv
-     (partial munge-subject summarize_by)
-     rows)))
+  (fn [_ _ _ _] 
```

Comment:
Actually @wkalt and I agreed it should probably be `(fn [_ _...] ...)` since eventually we'll probably use the `version` param again, i'll change them to be consistent.